tests/nightwatch/home.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  'Home page provides links to view new and processed credits' : function (browser) {
+    browser
+      .url('http://localhost:3000/')
```

Comment:
Did you intend for the port on this URL to be different to the one in `tests/nightwatch.json`?